cypress/elements/DimensionsPanel.js

```diff
@@ -0,0 +1,92 @@
+import { expectDimensionModalToBeVisible } from './DimensionModal'
+
+const dimButtonEl = 'dimensions-panel-list-dimension-item-button'
+const dimContextMenuButtonEl = 'dimensions-panel-list-dimension-item-menu'
+const dimContextMenuRemoveOptionEl =
+    'dimensions-panel-dimension-menu-item-remove'
+const dimContextMenuActionOptionEl =
+    'dimensions-panel-dimension-menu-item-action'
+const dimContextMenuSubMenuOptionEl =
+    'dimensions-panel-dimension-menu-item-DIMENSIONID-menu'
+const filterInputEl = 'dimensions-panel-filter'
+const fixedDimsWrapperEl = 'dimensions-panel-list-fixed-dimensions'
+const recommendedIconEl =
+    'dimensions-panel-list-dimension-item-recommended-icon'
+const dimSelectedBackgroundColor = 'rgb(224, 242, 241)'
+
+const getDimensionButtonById = dimensionId => `${dimButtonEl}-${dimensionId}`
+
+export const openContextMenu = dimensionId =>
+    cy.getBySel(`${dimContextMenuButtonEl}-${dimensionId}`).click()
+
+export const openDimension = dimensionId => {
+    cy.getBySel(getDimensionButtonById(dimensionId)).click()
+    expectDimensionModalToBeVisible(dimensionId)
+}
```

Comment:
These functions have similar names, but one does an assertion and the other doesn't. Maybe this should be consistent?

Reply:
Yeah, I agree, too much duplication, as mentioned here as well https://github.com/dhis2/data-visualizer-app/pull/1412/#discussion_r524959611. Will refactor this now.